### DISCUSSION TITLE
Will search be implemented soon?

### DISCUSSION THREAD

**Discussion by isle9** [jeffvli/feishin#58] [General]
The title.

**Comment by jeffvli** [comment]
Search is pretty high on my priority list.

It will likely be implemented before the next release.
Unfortunately I haven't had much time to work on it this month but I'm hoping sometime in the next 2 weeks.

**Reply by isle9** [reply]
I'm guessing this got moved to the next release?

edit: I recommend adding more status values in the project; for instance, high priority.

**Reply by jeffvli** [reply]
Yep, had to push this to next release.

**Comment by jeffvli** [comment]
Implemented in #107
